src/components/strings/uploadCopy.js

```diff
@@ -0,0 +1,19 @@
+export const desktopCopy = {
+  instructions: {
+    document: {
+      front: 'Upload front of document from your computer',
```

Comment:
This copy does not appear on the upload screen

Reply:
Oh yes, it was due to the merging